### ISSUE TITLE
file library.do not found

### ISSUE THREAD

**Issue by mcomsa** [carpenitoThomas/SuperLearner-STATA#1]
Hi,
SuperLearner is a wonderful idea. I tried to use your ado with your example on my PC, but I've got the following error: "file library.do not found (error occurred while loading superlearner.ado)". Please, advise. Thank you.

**Comment by carpenitoThomas** [comment]
Do you mind posting the call you make to the SuperLearner method?

**Comment by mcomsa** [comment]
Thank you. Please, see below.

clear
sysuse auto.dta
set seed 1
cd "C:\Users\Tommy\Documents\Berkeley\Thesis research"   // I have made this folder
*I tried both, with and without the option libraryglobals("library.do")
global custom_a = "regress mpg weight trunk price"
global custom_b = "regress mpg weight trunk"  
global custom_c = "regress mpg weight length" 
superlearner mpg length price weight length turn displacement,  k(10) family("gaussian") library("custom_a custom_b  custom_c regress ridgeregress lassoregress") superpredname("tommy") superestname("estimates") newdata("cars_altered.dta") originaldataset("auto.dta") libraryglobals("library.do")

/*
file library.do not found
(error occurred while loading superlearner.ado)
r(601);
*/

**Comment by carpenitoThomas** [comment]
You should create a do file (named "library.do") within the same directory. From there, you should include the global customs. If that does not work let me know and I'll take a further look at the issue.

**Comment by mcomsa** [comment]
Thank you so much. Now it is working. After I removed the examples section from the superlearner.ado, I can use it the from any folder. I will do some more tests with different datasets and learners and see what happens.
Considering the name of the ado, maybe the idea came from something similar implemented in R. Do you have any plans to continue the replication of the functionalities from R package SuperLearner or add some similar functions / options?

**Comment by mcomsa** [comment]
Hi again! Some more questions, if you have time, please. Thank you.
Regression problems:
What are the algorithms that are working (besides regress, ridgeregress, and lassoregress)?
I have tested so far ...
glm, sureg, elasticregress, and krls - are working
lasso2, npregress, bayes: reg, nl, and svmachines - are not working or I have done something wrong, maybe.

Classification problems:
Your example is not working. Neither mine. Please, see the file attached: 
[Test 1.zip](https://github.com/Tommyixi/SuperLearner-STATA/files/2364077/Test.1.zip)

**Comment by carpenitoThomas** [comment]
Absolutely, thank you for these findings! I will get back to you once I'm able to dive a bit deeper into what's going on.
